src/redux/utilities.js

```diff
@@ -18,3 +18,31 @@ export default function formatContentType(contentType) {
     return contentType;
   }
 }
+
+export function formatPublicationType(publicationType) {
+  let publicationTypes = {
+    all: 'All',
+    audiobook: 'Audiobook',
+    book: 'Book',
+    bookseries: 'Book Series',
+    database: 'Database',
+    journal: 'Journal',
+    newsletter: 'Newsletter',
+    newspaper: 'Newspaper',
+    proceedings: 'Proceedings',
+    report: 'Report',
+    streamingaudio: 'Streaming Audio',
+    streamingvideo: 'Streaming Video',
+    thesisdissertation: 'Thesis & Dissertation',
+    website: 'Web Site',
```

Comment:
Isn't "Website" more grammatically correct? (without a space)

I googled it and "Web Site" can be _technically_ correct (that's how it was spelled in the 90's), but most places I've seen the word, it's without a space

Reply:
Good point - I'll ask Khalilah about this - it is how the mapping is defined in the Jira ticket

Reply:
@wwilsman - Ok, confimed with Khalilah - this is a carryover from older legacy _90's_ app and she is onboard with your suggestion.  Thanks for recommendation - will make the changes -http://www.stratecomm.com/faqs/website/